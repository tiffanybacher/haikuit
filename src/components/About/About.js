import React from 'react';

function About() {
  return(
    <section className="About">
      <p>ABOUT</p>
    </section>
  );
}

export default About;