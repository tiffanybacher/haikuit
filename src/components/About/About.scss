.About {
  background: white;
  margin: 0;
  padding: 20px;

  .back-btn {
    font-style: italic;
    text-decoration: underline;
  }

  h2 {
    margin: 40px 0 20px 0;
  }
}

@media only screen and (min-width: 768px) {
  .About {
    font-size: 1.1em;
    padding: 50px;
  }
}

@media only screen and (min-width: 1000px) {
  .About {
    font-size: 1.2em;
    padding: 75px;
  }
}

@media only screen and (min-width: 1200px) {
  .About {
    font-size: 1.3em;
    padding: 100px 250px;
  }
}