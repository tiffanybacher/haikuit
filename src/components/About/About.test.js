import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe('About', () => {
  let mockHistory;
  let wrapper;

  beforeEach(() => {
    mockHistory = { goBack: jest.fn() };
    wrapper = shallow(
      <About history={mockHistory} />
    );
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});