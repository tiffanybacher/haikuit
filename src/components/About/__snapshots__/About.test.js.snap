// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match snapshot 1`] = `
<section
  className="About"
>
  <BackBtn
    goBack={[MockFunction]}
  />
  <h2>
    ABOUT
  </h2>
  <div
    className="About-blurb"
  >
    <p>
      <b>
        Welcome to Haikuit!
      </b>
    </p>
    <br />
    <p>
      Here at Haikuit, we know that it is natural to crave a creative outlet, but not always easy to find one that does not take somewhat of an investment in time and cash. We are here in hopes to spread the love for haiku. It is a free and easy way to let that creativity free!
    </p>
    <br />
    <p>
      Haiku is a traditional form of Japanese poetry. They consist of 3 lines. The content can anything! However, the first line is always 5 syllables, the second is 7 syllables, and the third is 5 syllables again. Haikus also rarely ryhyme.
    </p>
    <br />
    <p>
      Use Haikuit to easily check your syllables and to save all your haikus! Happy haikuing!
    </p>
  </div>
</section>
`;
